packages/components/src/icon/stories/index.story.tsx

```diff
@@ -7,44 +7,68 @@ import type { Meta, StoryFn } from '@storybook/react';
  * WordPress dependencies
  */
 import { SVG, Path } from '@wordpress/primitives';
-import { wordpress } from '@wordpress/icons';
+import * as icons from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
-import Icon from '..';
+import Icon, { type Props, type IconType } from '..';
 import { VStack } from '../../v-stack';
 
-const meta: Meta< typeof Icon > = {
+type IconProps = Props & {
+	additionalProps: object;
+};
+
+const meta: Meta< IconProps > = {
 	title: 'Components/Icon',
 	component: Icon,
 	parameters: {
 		controls: { expanded: true },
 		docs: { canvas: { sourceState: 'shown' } },
 	},
+	args: {
+		additionalProps: {},
+		size: 24,
```

Comment:
Without a value, when the user clicks the control to set a custom value, it defaults to 0, which seems worse than 24.

Reply:
I think we shouldn't neglect the importance of consistency and user expectations here. Not all components guarantee a static default. To have some controls default to an arbitrary value (which may not even be true) is confusing. If all number controls defaulted to zero, that can be more consistently understood as a meaningless placeholder value.

I think Storybook controls are designed to be used this way, given how certain control types require an actual button click ("Set number", "Set string", "Set boolean" etc.) to reveal the input. That step helps the user understand that the default input value has no meaning. Otherwise, maintainers of a Storybook instance would have to manually specify the default for every single control, which is not sustainable.